tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -362,3 +362,41 @@ def test_custom_tags(self):
         assert response.status_code == 200, response.content
         assert len(response.data) == 1
         self.assert_events_in_response(response, [event_1.event_id])
+
+    def test_wildcard_search(self):
```

Comment:
note to self: add some tests to check that regular expression generated by translate isn't too greedy (ex: `*sentry.io` won't match `*sentry.ioooooo`